backend/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    ports:
+      - "8001:5432"
+    volumes:
+      - data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: "postgres"
+      POSTGRES_DB: "fibo"
+    networks:
+      - pg-network
+  pgadmin:
+    image: dpage/pgadmin4
+    ports:
+      - "8080:80"
+    environment:
+      PGADMIN_DEFAULT_EMAIL: "aa@bb.cc"
+      PGADMIN_DEFAULT_PASSWORD: "aa@bb.cc"
+    networks:
+      - pg-network
+  django:
+    build: .
+    command: /usr/local/bin/gunicorn backend.wsgi:application -b :8000 --reload --max-requests 1
+    volumes:
+      - .:/code
+    ports:
+      - "8000:8000"
+    environment:
+      - POSTGRES_NAME=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
```

Comment:
This should be the same as in the .env file, but the Readme says tho create the .env File with different Variables

Reply:
I think the variable names are fine but we should use the .env file for the values. Something like

      - POSTGRES_NAME=${DB_NAME}


But we have to move the .env file to the folder where docker-compose.yml is locatet. Somehow adding the env_file tag in the yml is not enough to make it work from the given location